src/artifacts.js

```diff
@@ -446,6 +449,13 @@ api.declare({
     'stored externally. Either way, the response may not be JSON. So API',
     'client users might want to generate a signed URL for this end-point and',
     'use that URL with a normal HTTP client.',
+    '',
+    '**Caching**, artifacts may be cached in data centers closer to the',
+    'workers in-order to reduce bandwidth costs. This can lead to response',
+    'times at upwards 30 seconds. Caching can be skipped by setting the header',
```

Comment:
I don't think we should be putting a specific time in this comment.  Let's leave it as "This can lead to longer response times when there are problems with resource caching".